.yarn/versions/6ad01e88.yml

```diff
@@ -0,0 +1,3 @@
+releases:
+  "@radix-ui/react-checkbox": minor
+  primitives: minor
```

Comment:
Could you re-run the version check and `primitives` should be declined.
Also, I think it should just a be a patch, not a minor.